user-guide/docs/index.md

```diff
@@ -2,12 +2,14 @@
 
 ## User Account Registration
 
-Any natural hazards researcher or practitioner that wants an environment to store, analyze, curate, publish, and discover data with a community of peers may register for an account.
+Any natural hazards researcher or practitioner that wants an environment to store, analyze, curate, publish, and discover data with a community of peers may register for an account. Detailed account registration instructions can be found  [[here](https://docs.tacc.utexas.edu/basics/accounts/)].
 
 1. [**Request a user account**](https://www.designsafe-ci.org/account/register/)
-2. Follow instructions in the email you receive to complete setting up your account.
-3. You will then be able to [**log in to DesignSafe**](https://www.designsafe-ci.org/).<br />
-    (You will be required to use [multi-factor authentication](https://docs.tacc.utexas.edu/basics/mfa/).)
+2. You will be redirected to the TACC account registration page.
+3. You will first be asked to verify your identity via a 3rd party provider.
+4. On the account registration form we highly recommend you provide the email address provided by your Institution or Company.
+5. Once you have an account, set up [[multi-factor authentication](https://docs.tacc.utexas.edu/basics/mfa/)](https://docs.tacc.utexas.edu/basics/accounts/).
```

Comment:
Let's not double-nest link syntax, otherwise user would see:

> [&lsqb;multi-factor authentication&rsqb;(https://docs.tacc.utexas.edu/basics/mfa/)](https://docs.tacc.utexas.edu/basics/accounts/)

```suggestion
5. Once you have an account, set up [multi-factor authentication](https://docs.tacc.utexas.edu/basics/mfa/).
```